### PR TITLE
New fix

### PR DESCRIPTION
**Pull request by dys0n-V-nT**
I added a few new backgrounds/ aliases, fixed some commands, ill add some gifs for dance later

### CODE REVIEW

File: i18n/en-US/embedding/embedset.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "DESCRIPTION": "Begin an embed interface to create a custom embed live step by step.",
-  "EXTENDED": "**{{prefix}}embedset** ~ Begins an embed interface to create a custom embed live step by step.",
+  "EXTENDED": "$t(embedding/embedset:HELPER_MESSAGE_15)\n**{{prefix}}embedset** ~ Begins an embed interface to create a custom embed live step by step.",
```
Comment (Skillz4Killz) [inline i18n/en-US/embedding/embedset.json:3]: Please update this as similar to other commands that have multiple lines on EXTENDED. This breaks the pattern and makes it harder for users to translate who are beginners.

File: i18n/en-US/leveling/background.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	"EXTENDED_LINE_1": "**{{prefix}}background set [Background ID] [black or white]** ~ Sets a background and theme.",
 	"EXTENDED_LINE_2": "**{{prefix}}background view [Background ID] [black or white]** ~ View how a background and theme will look before setting it.",
 	"EXTENDED_LINE_3": "» To request a custom background, you must be a Nitro Booster of the Gamer support server. Use the invite command to join the server for more details.",
-	"EXTENDED_LINE_4": "» To see all the available backgrounds please check out our wiki page.",
+	"EXTENDED_LINE_4": "» To see all the available backgrounds please check out our wiki page. Background ID´s from 1 to 17.",
```
Comment (Skillz4Killz) [inline i18n/en-US/leveling/background.json:7]: IMO you can remove the part about checking it on the wiki.

File: src/commands/fun/dance.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { Command } from 'yuuko'
```
Comment (Skillz4Killz) [inline src/commands/fun/dance.ts:1]: See if u can make a mission for this command as well

File: src/commands/fun/bite.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { Command } from 'yuuko'
 import GamerClient from '../../lib/structures/GamerClient'
 import constants from '../../constants'
 
-export default new Command(`bite`, async (message, _args, context) => {
+export default new Command([`bite`, `chomp`], async (message, _args, context) => {
```
Comment (Skillz4Killz) [inline src/commands/fun/bite.ts:5]: make a mission for this command as well

File: src/commands/roles/roleinfo.ts [hunk 5]
```diff
@@ -16,10 +16,17 @@ export default new Command([`roleinfo`, `ri`], async (message, args, context) =>
 
   const language = Gamer.getLanguage(message.guildID)
 
+  if (!Gamer.allMembersFetchedGuildIDs.has(message.member.guild.id)) {
```
Comment (Skillz4Killz) [inline src/commands/roles/roleinfo.ts:19]: Only do this on VIP guilds

File: src/constants/gifs.ts [hunk 6]
```diff
@@ -567,5 +567,7 @@ export default {
     'https://media.tenor.com/images/f4a52ebfaec15c4e8bace6cb141d72f9/tenor.gif',
     'https://media.tenor.com/images/8409bd65be28e7bcc5a7630c4ebbdcca/tenor.gif',
     'https://media.tenor.com/images/6ab39603ef0dd6dbfc78ba20885b991f/tenor.gif'
+  ],
+  dance: [
```
Comment (Skillz4Killz) [inline src/constants/gifs.ts:571]: make sure to add some gifs here for servers that like to disable tenor api

File: src/commands/roles/roleinfo.ts [hunk 7]
```diff
@@ -20,7 +20,7 @@ export default new Command([`roleinfo`, `ri`], async (message, args, context) =>
     .setAuthor(role.name, message.author.avatarURL)
     .addField(language(`roles/roleinfo:ROLE_NAME`), role.mention, true)
     .addField(language(`roles/roleinfo:ROLE_ID`), role.id, true)
-    .addField(language(`roles/roleinfo:ROLE_COLOR`), `#${role.color.toString(16).toUpperCase()}`, true)
+    .addField(language(`roles/roleinfo:ROLE_COLOR`), `${role.color.toString(16).toUpperCase()}`, true)
```
Comment (Skillz4Killz) [inline src/commands/roles/roleinfo.ts:23]: Why did you remove the # here?

File: i18n/en-US/leveling/missions.json [hunk 8]
```diff
@@ -22,5 +22,6 @@
   "MINES": "Play a game of Minesweeper",
   "LMAO": "Share a laugh with others with the .lmao command.",
   "SLOTS": "Play a game of slots to earn Gamer Coins with .slots command.",
-  "WALLPOST": "Make a post on your wall channel. To get a wall channel use the .networkcreate command."
-}
+  "WALLPOST": "Make a post on your wall channel. To get a wall channel use the .networkcreate command.",
+  "BITE": "Bite 5 people."
```
Comment (Skillz4Killz) [inline i18n/en-US/leveling/missions.json:26]: Can u make sure to to translate this in pt-br also. Just use google translate. The pt-br servers use missions 24/7 so its important to translate missions for them

Comment (dys0n-V-nT) [reply i18n/en-US/leveling/missions.json:26]: This get transalted if u translate it

File: src/commands/roles/roleinfo.ts (hunk 7 above)
Comment (dys0n-V-nT) [reply src/commands/roles/roleinfo.ts:23]: I thought it was a typo, so i changed that back

File: src/commands/fun/bite.ts (hunk 4 above)
Comment (dys0n-V-nT) [reply src/commands/fun/bite.ts:5]: mission done

File: src/commands/roles/roleinfo.ts (hunk 5 above)
Comment (dys0n-V-nT) [reply src/commands/roles/roleinfo.ts]: only for vip done

File: src/constants/gifs.ts (hunk 6 above)
Comment (dys0n-V-nT) [reply src/constants/gifs.ts:571]: added gifs

**Comment by Skillz4Killz** [comment]
Very impressive good stuff!
